cms/djangoapps/contentstore/views/course.py

```diff
@@ -1031,19 +1028,16 @@ def settings_handler(request, course_key_string):
 
             about_page_editable = not marketing_site_enabled
             enrollment_end_editable = GlobalStaff().has_user(request.user) or not marketing_site_enabled
-<<<<<<< HEAD
-            short_description_editable = settings.FEATURES.get('EDITABLE_SHORT_DESCRIPTION', True)
 
-            default_enroll_email_template_pre = render_to_string('emails/default_pre_enrollment_message.txt', {})
-            default_enroll_email_template_post = render_to_string('emails/default_post_enrollment_message.txt', {})
-
-=======
             short_description_editable = configuration_helpers.get_value_for_org(
                 course_module.location.org,
                 'EDITABLE_SHORT_DESCRIPTION',
                 settings.FEATURES.get('EDITABLE_SHORT_DESCRIPTION', True)
             )
->>>>>>> 90707afa503dfba74c592f88ce43c01d12c76142
+            
+            default_enroll_email_template_pre = render_to_string('emails/default_pre_enrollment_message.txt', {})
+            default_enroll_email_template_post = render_to_string('emails/default_post_enrollment_message.txt', {})
+
```

Comment:
split out to it's own file, add to context below with:
`settings_context.update(_get_stanford_context())`